tools/print_config.py

```diff
@@ -12,18 +13,44 @@ def parse_args():
     parser.add_argument(
         '--graph', action='store_true', help='print the models graph')
     parser.add_argument(
-        '--options', nargs='+', action=DictAction, help='arguments in dict')
+        '--options',
+        nargs='+',
+        action=DictAction,
+        help='override some settings in the used config, the key-value pair '
+        'in xxx=yyy format will be merged into config file. If the value to '
+        'be overwritten is a list, it should be like key="[a,b]" or key=a,b '
+        'It also allows nested list/tuple values, e.g. key="[(a,b),(c,d)]" '
+        'Note that the quotation marks are necessary and that no white space '
+        'is allowed.')
```

Comment:
It would be better to add '--options is deprecating in favor of --cfg_options' in the help message, as it will make users confused when they see there are two different arguments but have the exactly same description.